### ISSUE TITLE
Comply without acknowledgement

### ISSUE THREAD

**Issue by PeterMurdoch** [d8ahazard/Phlex#255]
I wired a shortcut so my toddler can say hey Google it's too late and Carole King plays.  The shortcut works well, but it would be nice to eliminate as much of "ok I'll ask flex tv for this, ok playing it's too late" as possible, at least once a shortcut has been finalized.  Any tips?

**Comment by d8ahazard** [comment]
Using actions, there isn't any way to stop the default "conversation
flow".  It's just how Google says it has to be.

However, you could set up a command in IFTTT where you can customize both
the invocation and response.  I'm not sure if it'll allow no response at
all, but you could definitely whittle it down to just "ok".

**Comment by PeterMurdoch** [comment]
Hmm, I see.  Ok, I can revisit IFTTT I had started there but it was not successful.  I'll circle back if I am further blocked.

Thanks for your work here, it is the only way I have found to get a Google home device to play audio/video on my local network. 
I am looking into whether this project would also be an option to remove the plex dependency for directly reading dlna and wonder if there could be any synergies - https://github.com/openhab/openhab-google-assistant
